content/waku/2024-gantt.md

```diff
@@ -0,0 +1,180 @@
+%% Research Team
+    section (rsrch) - jm-clius
+        White Paper: whitepaper, 2024-01-01, 2024-12-01
+        %% TBC end
+        RFC review: rfc-review, 2024-01-01, 90d
+        %% TBC start and estimation
+        Status Communities protocol scaling recommendation: status-proto-r, after rfc-review, 12w
+    section (rsrch) alrevuelta
+        %% TBC that work is done
+        RLN in resource-restricted clients: rlnrestricted, 2024-01-01, 110d
+        %% TBC estimation
+        RLNv2 in nwaku: rlnv2-n, 2024-02-01, 18w
+        Maturing RLN roadmap: maturing-rln-roadmap, 2024-01-01, 2024-12-01
+    section (rsrch) sionois
+        %% TBC estimation done?
+        Store v3-beta (msg hash): crit, storev3-br, 2024-01-01, 12w
+        %% TBC estimation
+        Store v3 (sync): crit, storev3-r, after storev3-br, 20w
+        %% TBC estimation
+        Active Capability Service Discovery PoC: act-cap-disc-r, after storev3-r, 30w
+        %% TBC estimate allocation
+        Review usage of content topics: rev-content-topics, after act-cap-disc-r, 8w
+    section (rsrch) s-tikhomirov
+        %% TBC start and estimation
+        Pay for RLN provision first PoC: pay-rln-r, 2024-02-01, 20w
+    section (rsrch) shash256
+        %% TBC estimate/end
+        Provision RLN for light push clients PoC: provision-rln-r, 2024-04-28, 5w
+        %% TBC estimate
+        Multicast end-to-end reliability protocol - PoC: m-e2e-rel-r, after provision-rln-r, 20w
+%% nwaku team
+    section (nwak) ivansete
+        PostgreSQL Maintenance: crit, pgsql, 2024-01-01, 2024-05-31
+        %% TBC estimate, this is assuming PoC is done, ie,  how much hardening would be needed?
+        Store v3-beta + v3 (nwaku): crit, storev3-n, after storev3-br, 3w
+        %% TBC estimate
+        Reliability Protocol for Relay (nwaku): crit, rel-relay-n, 2024-05-01, 40w
+    section (nwak) NagyZoltanPeter
+        %% TBC estimate/end
+        DoS Protection for Req-Res Protocols: crit, dosreqresn, 2024-01-01, 90d
+        %% TBC estimate
+        Usage of rendezvous: crit, rendezvous-n, after dosreqresn, 4w
+        %% TBC Estimate
+        Nwaku in Golang desktop: nwaku-golang-desk-n, after rendezvous-n, 2w
+        %% TBC Estimate
+        Nwaku in Golang - relay: nwaku-golang-relay-n, after nwaku-golang-desk-n, 4w
+        %% TBC Estimate
+        Nwaku in Status Desktop (Relay, *nix): nwaku-status-d-n, after nwaku-golang-relay-n, 8w
+    section (nwak) gabrielmer
+        %% TBC estimate/end
+        Sharding peer mgmt and discovery hardening (nwaku): crit, sh-peer-mgmt-n, 2024-05-13, 12w
+    section (nwak) d
+        (telemetry) Fleet logging: telem-fleet-logging, 2024-05-15, 4w %% TBC estimate/end/start
+%% js-waku team
+    section (jwak) weboko
+        %% TBC now stopped
+        RLN in resource-restricted clients: rlnlight, 2024-01-01, 90d
+        %% TBC now stopped
+        Local Web Development: localweb, 2024-02-01, 30d
+        %% TBC status
```

Comment:
asking for clarification on what `status` means?

Reply:
what is the status of this?